main.go

```diff
@@ -6,45 +6,68 @@ package main
 
 import (
 	"fmt"
-	"log"
+	"os"
 
 	"github.com/docopt/docopt-go"
 	"github.com/taskcluster/taskcluster-worker/engines/extpoints"
+	"github.com/taskcluster/taskcluster-worker/runtime"
 )
 
 const version = "taskcluster-worker 0.0.1"
 const usage = `
-Usage: taskcluster-worker [options]
-Runs a worker with the given options.
-Options:
-  -V, --version            Display the version of go-import-subtree and exit.
-  -h, --help               Print this help information.
-  -e, --engine <engine>    Execution engine to run tasks in
+TaskCluster worker
+This worker is meant to be used with the taskcluster platform for the execution and
+resolution of tasks.
+
+  Usage:
+    taskcluster-worker --help
+    taskcluster-worker --version
+    taskcluster-worker --engine <engine>
+    taskcluster-worker --engine <engine> --logging <level>
+
+  Options:
+    --help  						Show this help screen.
+    --version  						Display the version of go-import-subtree and exit.
+    -e --engine <engine>  			Engine to use for task execution sandboxes.
+    -l --logging-level <level>  	Set logging at <level>.
 `
 
 func main() {
 	args, err := docopt.Parse(usage, nil, true, version, false, true)
 	if err != nil {
-		log.Fatalf("Error parsing arguments. %v", err)
+		fmt.Fprintf(os.Stderr, "Error parsing arguments. %v", err)
+		os.Exit(1)
 	}
 
-	e := args["--engine"]
-	if e == nil {
-		panic("Must supply engine type")
+	var level string
+	if l := args["--logging-level"]; l != nil {
+		level = l.(string)
+	}
+	logger, err := runtime.CreateLogger(level)
+	if err != nil {
+		os.Stderr.WriteString(err.Error())
+		os.Exit(1)
```

Comment:
I think `panic` is appropriate for crashes, but not appropriate for handling bad input - i.e. a user should never see a stack trace because he/she passed in an unrecognised logging level. In any case, I think we should process the underlying cause and crash if really an internal error, or report the bad value, if it is just a bad value. We could also consider creating and documenting different exit codes for different failures, although I believe exit code 1 is appropriate for invalid arguments.